### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by Keytus**

### CODE REVIEW

**Review by nastik1811 — COMMENTED** [review]
@Keytus 🤔 
image

**Comment by Keytus** [comment]
Fixed.Could you check it, please?

**Comment by nastik1811** [comment]
Something wrong with mobile version
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            </button>
+        </form>
+    </div>
+    
+
+    <script>
+        {
+            var currentForm=null;
+            var isOpen = false;
+            var canClose = false;
```
Comment (nastik1811) [inline index.html]: Don't use `var`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                        </li>
+                        <li>
+                            <a class="navigation-panel-genres__item navigation-panel-genres__item_font" href="#">Genre 6</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="bookcase">
+                <div class="audio-items">
+                    <figure type="button" class="audio-item" onclick="openForm('PlayerForm')">
```
Comment (nastik1811) [inline index.html]: ` type="button" `? I don't remember this attribute on `figure`.

Comment (nastik1811) [reply index.html]: I also think it was a doubtful idea to use figure in the way you did
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+<body>
+    <div id="page" style="height: inherit;" onclick="closeCurrentForm()">
+        <header class="header">
+            <span class="header__websitename header__websitename_font">Audio-Library</span>
+            <div class="button-container">
+                <form>
+                    <button type="button" class="button-container__auth-button button-container__auth-button_standard" onclick="openForm('LoginForm')">
+                        Login
+                    </button>
+                </form>
```
Comment (nastik1811) [inline index.html]: Why have you wrapped this in `form`?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+<head>
+    <title>Audio-Library</title>
+    <link rel="stylesheet" href="common.blocks/desktop/main/main.css" media="screen and (min-width : 361px)">
+    <link rel="stylesheet" href="common.blocks/mobile/main/main.css" media="screen and (max-width: 360px)">
+    <link rel="stylesheet" href="common.blocks/fix-screen/fix-screen.css">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">  
+    <meta charset="utf-8" content="width=device-width" name="viewport" >
+</head>
+<body>
+    <div id="page" style="height: inherit;" onclick="closeCurrentForm()">
```
Comment (nastik1811) [inline index.html]: `style="height: inherit;` ?

Comment (nastik1811) [reply index.html]: Why do you need inline styles here?

File: common.blocks/desktop/popup/login-form/login-form.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.login-form{
+    display: none;
+    flex-direction: column;
+    width: 800px;
+    height: 350px;
+    position: absolute;
+    top: calc(100% / 2 - 175px);
+    left: calc(100% / 2 - 400px);
+}
```
Comment (nastik1811) [inline common.blocks/desktop/popup/login-form/login-form.css:9]: You have the same styles for .sign-up form. It's better to use common class to style them (btw, you have no reason to set absolute height for them)

File: common.blocks/desktop/popup/add-form/add-top/add-top.css [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    height: 70px;
+    width: 600px;
+    background-color: #9D9D9D;
+    align-items: center;
+}
+
+.add-top__websitename {
+    margin-left: 30px;
+    width: 210;
+    height: 40;
```
Comment (nastik1811) [inline common.blocks/desktop/popup/add-form/add-top/add-top.css]: what about units?

File: common.blocks/desktop/popup/add-form/add-bottom/add-bottom.css [hunk 7]
```diff
@@ -0,0 +1,43 @@
+.add-bottom{
+    display: inherit;
+    flex-direction: column;
+    height: 330px;
+    min-height: 330px;
```
Comment (nastik1811) [inline common.blocks/desktop/popup/add-form/add-bottom/add-bottom.css]: `min-height` has no sense, since `height` set absolutely

**Comment by nastik1811** [comment]
image

File: common.blocks/desktop/footer/footer.css [hunk 8]
```diff
@@ -0,0 +1,19 @@
+.footer {
+    display: flex;
+    flex-direction: row-reverse;
+    background-color: #A2A2A2;
+    align-items: center;
+    width: 1440px;
```
Comment (nastik1811) [inline common.blocks/desktop/footer/footer.css]: you should not use absolute width/height for full-width/height containers

**Review by nastik1811 — CHANGES_REQUESTED** [review]
There is still something wrong with your styles

**Comment by Keytus** [comment]
Remove var,remove type="button" from figures and buttons, unwraped buttons from form, remove inline styles, replace .sign-up and .login forms with .auth form, remove redundant size properties. Mobile version: it activated when device width 360px or less. Figures: i just wanna unite .mp3 image and .mp3 info as logically indivisible unit of content.

File: index.html [hunk 9]
```diff
@@ -164,24 +156,20 @@
...
     
 
     <script>
         {
-            var currentForm=null;
-            var isOpen = false;
-            var canClose = false;
+            currentForm=null;
+            isOpen = false;
+            canClose = false;
```
Comment (nastik1811) [inline index.html]: image

Comment (nastik1811) [reply index.html]: image

**Review by nastik1811 — APPROVED** [review]
You should use `let` or `const` in your js

For this work you get `7`

**Pull request merged**